flight/PiOS/STM32F4xx/inc/pios_dac.h

```diff
@@ -0,0 +1,89 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS_DAC DAC Functions
+ *
+ * @file       pios_dac.h
+ * @author     dRonin, http://dronin.org Copyright (C) 2017
+ * @see        The GNU Public License (GPL) Version 3
+ *
+ *****************************************************************************/
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ * Additional note on redistribution: The copyright and license notices above
+ * must be maintained in each individual source file that is a derivative work
+ * of this source file; otherwise redistribution is prohibited.
+ */
+
+#ifndef _PIOS_DAC_H
+#define _PIOS_DAC_H
+
+#include <stdint.h>
+#include <stdio.h>
+
+#include "stm32f4xx_dac.h"
+
+typedef struct dac_dev_s *dac_dev_t;
```

Comment:
Last several drivers we merged like ws2811, max7456, lis3mdl we had decided not to do that and to move towards changing the convention.   Want to revisit?